packages/usa-form-group/src/styles/_usa-form-group.scss

```diff
@@ -11,11 +11,11 @@
 
 // Block input elements
 .usa-form-group--error {
-  @include u-border-left(0.5, "error-dark");
-  padding-left: units(2);
+  @include u-border-left(2px, "error-dark");
+  padding-left: units(1);
   position: relative;
 
   @include at-media("desktop") {
-    margin-left: units(-2.5);
+    margin-left: -10px;
```

Comment:
Matching the existing behaviors, this number is equal to the negation of border width + padding. Here I'm using a magic number but we could make the styles smarter by assigning variables and calculating the `margin-left`.

```scss
$form-error-border-width: 2px;
$form-error-border-padding: rem-to-px(units(1)); // Alternatively, we could just set this to 8px
$form-error-negative-margin: -(
  $form-error-border-width + $form-error-border-padding
);

...

.usa-form-group--error {
  @include u-border-left($form-error-border-width, "error-dark");
  padding-left: $form-error-border-padding;
  position: relative;

  @include at-media("desktop") {
    margin-left: $form-error-negative-margin;
  }
}